raster-foundry/raster-foundry
# Fix AOI list filter and other aoi bugs

## Overview

This PR:

- fixes the aoi list filter on the projects endpoint to use the project id
- removes reference to `aois_to_projects` from the `FindAOIProjects` batch job
- removes the `AoiToProjectDao`

### Checklist


## Notes

~wip until a test is added that confirms that project filtering is happening in aoi listing for a project~ c382056

## Testing Instructions

 * CI
 * fire up your server and frontend
 * create two AOI projects
 * manually (in the db) set their last checked times to jan 1 1900 and jan 1 2300
 * open up a console in the batch subproject
 *
```scala
import com.azavea.rf.database.util.RFTransactor.xa
import com.azavea.rf.batch.aoi.FindAOIProjects
FindAOIProjects().run
```
 * you should see the project id for the project with the last checked date in the past
